HW1/SoftServeWOstl/FileWithoutSTL.cpp

```diff
@@ -0,0 +1,135 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <cstdlib>
+#include <conio.h>
+using namespace std;
+
+void Generate(string arr[]) {
+	const int size = 5000;
+	string arr2[ size ];
+	for (int i = 0; i < size;++i) {
+		arr2[i] = arr[rand() % 10];
+	}
+
+	const char* Fname = "Name.txt";
+	ofstream fout(Fname, ios::app);
+	if (!fout.is_open()) {
+		cout << "Error open";
+	}
+	else {
+		for (int i = 0; i < size; ++i) {
+			fout << arr2[i];
+		}
+
+	}
+	fout.close();
+}
+
+int weight(string word) {
+	int count = 0;
+	for (int i = 0; i < strlen(word.c_str()); i++) {
+		if (word[i] == 'a')count++;
+		if (word[i] == 'b')count+=2;
```

Comment:
you are so NOT lazy to write all of this))

Reply:
)